scout/parse/matchmaker.py

```diff
@@ -158,6 +159,18 @@ def genomic_features(store, case_obj, sample_name, candidate_vars, genes_only):
     return g_features
 
 
+def parse_datetime(match_date: Union[str, int]) -> datetime:
+    """Converts a date to a datetime."""
```

Comment:
I would suggest moving the comments from below up here to the docstring. They would be useful as an explanation to the different formats.